docs/src/pages/index.tsx

```diff
@@ -1,37 +1,259 @@
+            The entire configuration process happens over HTTP API, and streams are delivered over
+            RTP. You can leverage technology of your choice and easily integrate it with your own
+            solutions.
+          </p>
+        </ProsCard>
+        <ProsCard title="Hardware accelerated" icon={FaMicrochip}>
+          <p className="padding--md">
+            Rendering is GPU accelerated using {wgpuLink} library, which implements API based on the
+            WebGPU standard. Parts of the standard, like the ability to run WGSL shaders, are
+            directly accessible via API.
+          </p>
+        </ProsCard>
+      </div>
+    </div>
+  );
+}
+
+type ProsCardProps = {
+  title: string;
+  icon: IconType;
+};
+
+function ProsCard(props: PropsWithChildren<ProsCardProps>) {
+  const Icon = props.icon;
+  return (
+    <div className={clsx('card', styles.card)}>
+      <div style={{ display: 'flex', justifyContent: 'center' }}>
+        <Icon className={styles.icon} />
+      </div>
+      <div className="card__header">
+        <Heading as="h3" style={{ textAlign: 'center' }}>
+          {props.title}
+        </Heading>
+      </div>
+      <div className="card__body">{props.children}</div>
+    </div>
+  );
+}
+
+type FeatureProps = {
+  text: string;
+  secondaryText?: string;
+  image: any;
+  inverted?: boolean;
+};
+
+function Feature(props: FeatureProps) {
+  const text = (
+    <div className="col">
+      <Heading as="h2" className="margin-top--sm">
+        {props.text}
+      </Heading>
+      <p>{props.secondaryText}</p>
+    </div>
+  );
+  if (props.inverted) {
+    return (
+      <div className="row margin-right--lg margin-left--lg" style={{ alignItems: 'center' }}>
+        {props.image}
+        {text}
+      </div>
+    );
+  } else {
+    return (
+      <div className="row margin-right--lg margin-left--lg" style={{ alignItems: 'center' }}>
+        {text}
+        {props.image}
+      </div>
+    );
+  }
+}
+
+function Features() {
+  const { colorMode } = useColorMode();
+  const imageTextRendering = (
+    <div className="row margin--lg">
+      <div>
+        <FaImage style={{ fontSize: 80, marginBottom: 30 }} />
+      </div>
+      <div>
+        <FaFont style={{ fontSize: 80, marginTop: 30 }} />
+      </div>
+    </div>
+  );
+  const wgpuLogo =
+    colorMode === 'dark' ? (
+      <WebGpuLogoDark className="margin--lg" width={100} height={100} />
+    ) : (
+      <WebGpuLogoLight className="margin--lg" width={100} height={100} />
+    );
+
+  return (
+    <div className="container margin-top--lg margin-bottom--md">
+      <Heading as="h2" className="margin-bottom--md">
+        Features
+      </Heading>
+
+      <Feature
+        text="WebGPU APIs"
+        secondaryText="Use existing WebGPU-based components provided by the compositor. If you need anything not covered by the existing components, you can always write your own WGSL shader that implements that effect."
```

Comment:
```suggestion
        secondaryText="Utilize existing WebGPU-based components provided by the compositor or craft your own WGSL shader to achieve any desired effect."
```

IMO making it shorter makes the description more interesting